spec/core/ics-004-channel-and-packet-semantics/README.md

```diff
@@ -84,7 +84,7 @@ interface ChannelEnd {
 - The `nextSequenceSend`, stored separately, tracks the sequence number for the next packet to be sent.
 - The `nextSequenceRecv`, stored separately, tracks the sequence number for the next packet to be received.
 - The `nextSequenceAck`, stored separately, tracks the sequence number for the next packet to be acknowledged.
-- The `connectionHops` stores the list of connection identifiers, in order, along which packets sent on this channel will travel. At the moment this list must be of length 1. In the future multi-hop channels may be supported.
+- The `connectionHops` stores the list of connection identifiers, in order, along which packets sent on this channel will travel. A list length is greater than 1 indicates a multi-hop channel.
```

Comment:
`connectionHops` appears in `ChanOpenInit` and `ChanOpenTry`. I think we should clarify that the order is starting from the receiving chain towards the other end, i.e. `connectionHops[0]` is the end on the chain receiving this message. 

In this example:
`A (a) --- (b1) B (b2) --- (c) C`
for `OpenInit` we will have `[a, b2]`
for `OpenTry` we will have `[c, b1]`

See also this line that appears in `ChanOpenInit` and `ChanOpenTry`:
`connection = provableStore.get(connectionPath(connectionHops[0]))`

I would also suggest doing the same for proofs, i.e. reverting the order in the proof collection pseudocode (`prepend` instead of `append`) so they appear in the same order as the `connectionHops`.
At the moment in `VerifyConnectionHops` it seems that the `connectionProofs` are in the reverse order of the `connectionHops`.

Alternatively we can change the `ChanOpenTry` to have the `connectionHops` in this order in the example:
`[c, b1]`
and get the connection like this:
`connection = provableStore.get(connectionPath(connectionHops[n-1]))`
(lower preference for me)

Reply:
No problem, added some info on the connectionHops and updated the proof ordering per your suggestion.